std_msgs/Header header
MaskData[] mask_data
kinect_pub/RegistrationData registration_data